spec/support/es_spec_helper.rb

```diff
@@ -56,6 +56,8 @@ def create_es_index(klass)
     create_es_index(Manifest)
   end
   config.after :suite do
-    stop_es_server
+    if es_server_running?
```

Comment:
the check for `es_server_running?` already happens inside `stop_es_server` -- any reason you needed to check 2x?